### PR TITLE
Adding limitSpec support to groupby query

### PR DESCRIPTION
**Pull request by mistercrunch**

### CODE REVIEW

File: pydruid/client.py [hunk 1]
```diff
@@ -442,13 +445,18 @@ def groupby(self, **kwargs):
             :linenos:
 
                 >>> group = query.groupby(
-                        dataSource='twitterstream',
+                        datasource='twitterstream',
```
Comment (mistercrunch) [inline pydruid/client.py]: This edit is unrelated to this PR, but this fixes the doctest

**Comment by xvrl** [comment]
Thanks for the contribution @mistercrunch, do you mind signing our CLA (http://druid.io/community/cla.html) so I can merge?

**Comment by mistercrunch** [comment]
Done!

**Comment by mistercrunch** [comment]
img

**Pull request merged**